nextcloud_update.sh

```diff
@@ -309,7 +309,16 @@ fi
 # Update all Nextcloud apps
 if [ "${CURRENTVERSION%%.*}" -ge "15" ]
 then
-    occ_command app:update --all
+    UPDATED_APPS=$(occ_command app:update | sed '1~2d' | awk '{print $1}' | tr '\n' " ")
+    UPDATED_APPS=${UPDATED_APPS// /", "}
+fi
+
+# Check which apps got updated
+if [[ "$UPDATED_APPS" == @([a-z]) ]]
```

Comment:
this checks whether updated_apps contains a a-z character, which should always be the case if apps got updated.